doc: minimal documentation of supported platforms (#254741)
NixOS/nixpkgs: doc/using/platform-support.chapter.md

# Platform Support {#chap-platform-support}

Below is a list of well-supported supported platforms, in terms of both CI resources and developer attention:

- `x86_64-linux`: Highest level of support.
- `aarch64-linux`: Well supported, with most packages building successfully in CI.
- `aarch64-darwin`: Receives better support than `x86_64-darwin`.
- `x86_64-darwin`: Receives some support.

> Many other platforms are supported, with varying levels of support.

Author: Thanks, I was looking for a way out of that one :D

# Platform Support {#chap-platform-support}

Packages receive varying degrees of support, both in terms of maintainer attention and available computation resources for continuous integration (CI). 

> Below is a list of well-supported supported platforms:

Reviewer: ```suggestion
Here is the list of platforms with some support:
```

Reviewer: Either sounds good!
